touchforms/formplayer/api.py

```diff
@@ -320,9 +320,35 @@ def post_data(data, auth=None, content_type="application/json"):
     if auth:
         d['hq_auth'] = auth.to_dict()
     # just default to old server for now
+
+    if toggles.SMS_USE_FORMPLAYER.enabled():
```

Comment:
need to pass the domain in too.

would be good to do a run through of the test suite and confirm that nothing breaks with these changes